server/controller/answers.js

```diff
@@ -0,0 +1,22 @@
+const Question = require('../models/question.js');
+const Answer = require('../models/answer.js');
+
+exports.addAnswer = async (ctx) => {
+  await Question.findById(ctx.request.body.question_id)
+    .then((res) => {
+      if (res) {
+        const answer = new Answer(ctx.request.body);
+        return answer.save();
+      }
+
+      ctx.body = false;
```

Comment:
Instead of returning `false`,  we could return an object with an error message so we know that the `question_id ` doesn't exist.